app/api.py

```diff
@@ -38,23 +40,28 @@ def signin():
         data = request.get_json()  # get data from postman
         email = data['email']  # get email
         password = data['password']  # get password
-        userdata = db.session.query(user_data).filter_by(
+        userdata = db.session.query(users.Users).filter_by(
```

Comment:
model name should be defined as a singular entity.. i.e. User instead of Users

Reply:
Changed